policy/lib/tekton/pipeline.rego

```diff
@@ -4,6 +4,7 @@ import future.keywords.contains
 import future.keywords.if
 import future.keywords.in
 
+import data.lib
 import data.lib.time
 
 pipeline_label := "pipelines.openshift.io/runtime"
```

Comment:
The label in the task is `build.appstudio.redhat.com/build_type`. This should be updated to that. 

An example.
https://github.com/redhat-appstudio/build-definitions/blob/main/task/buildah/0.1/buildah.yaml#L6

Reply:
Thank you! In the process, should I remove support for looking at the labels on the slsa provenance for the pipeline, and only look at the labels for the task?

Reply:
I kept it since it's just used as a fallback.